.changeset/empty-waves-look.md

```diff
@@ -0,0 +1,5 @@
+---
+'urql-docs': patch
+---
+
+docs: remove double negation
```

Comment:
You don't need to add a changeset for docs as we can't publish these

Reply:
Removed :)